specs/laravel/laravel_event_clear.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: Clear laravel events and listeners

```

Comment:
nit: could we be consistent with uppercase for lowercase for `Laravel`? Maybe always uppercase?

Reply:
Sure. On it now